exampleCourse/questions/element/pythonVariable/question.html

```diff
@@ -5,14 +5,11 @@
 </pl-question-panel>
 
 <div class="card my-2">
+  <!-- TODO remove this DataFrame stuff once we test that it looks as it did before, so we can fully deprecate the pandas part of this element. -->
```

Comment:
Will remove before a merge, but leaving this in just to test that the passthrough works and output looks the same.

Reply:
Has this been thoroughly tested now?

Reply:
Yes, my previous tests all actually worked, I just left this in case there was a large code change and I wanted to make sure the pass-through still worked. Since the interface is fine, I'll go ahead and remove.

Reply:
Tested again and is working!